src/shared/components/Loader.tsx

```diff
@@ -32,3 +32,5 @@ export default (props: LoaderProps) => {
   }
   return <></>;
 };
+
+export default Loader
```

Comment:
Please add newline and semicolon

Reply:
https://github.com/UWmathNEWS/slugline-web/pull/85/commits/565f4fa3bd218e550b30248004596be5f6030d99